CURRENT_SUPPORTED_VERSIONS

```diff
@@ -1,4 +1,3 @@
+2.12
 2.11
 2.10
-2.9
-2.8
```

Comment:
Let's leave 2.9 in the list. 2.8 support is dropped in December, but 2.9 is supported until next June:
- https://access.redhat.com/support/policy/updates/advanced-cluster-management

Reply:
Thanks!!